Bio/_utils.py

```diff
@@ -36,7 +36,7 @@ def iterlen(items):
 
 
 def read_forward(handle):
-    """Reads through whitespaces, returns the first non-whitespace line."""
+    """Read through whitespaces, returns the first non-whitespace line."""
```

Comment:
Should probably be "Read through whitespaces, return the first non-whitespace line."